plugins/aws/connect/instanceMediaStreamsEncrypted.js

```diff
@@ -0,0 +1,132 @@
+                    helpers.addResult(results, 3,
+                        `Unable to describe Connect instance media streams storage config: ${helpers.addError(listInstanceMediaStreamsStorageConfigs)}`,
+                        region, resource);
+                    continue;
+                }
+
+                if (!listInstanceMediaStreamsStorageConfigs.data.StorageConfigs.length) {
+                    helpers.addResult(results, 0,
+                        'Connect instance does not have any storage config for media streams',
+                        region, resource);
+                    continue;
+                }
+
+                let storageConfig = listInstanceMediaStreamsStorageConfigs.data.StorageConfigs[0];
+
+                if (storageConfig.KinesisVideoStreamConfig) {
+                    if (storageConfig.KinesisVideoStreamConfig.EncryptionConfig &&
+                        storageConfig.KinesisVideoStreamConfig.EncryptionConfig.KeyId) {
+                        let kmsKeyArn = storageConfig.KinesisVideoStreamConfig.EncryptionConfig.KeyId;
+                        let keyId = kmsKeyArn.split('/')[1] ? kmsKeyArn.split('/')[1] : kmsKeyArn;
+
+                        var describeKey = helpers.addSource(cache, source,
+                            ['kms', 'describeKey', region, keyId]);  
+    
+                        if (!describeKey || describeKey.err || !describeKey.data || !describeKey.data.KeyMetadata) {
+                            helpers.addResult(results, 3,
+                                `Unable to query KMS key: ${helpers.addError(describeKey)}`,
+                                region, kmsKeyArn);
+                            continue;
+                        }
+    
+                        currentEncryptionLevel = helpers.getEncryptionLevel(describeKey.data.KeyMetadata, helpers.ENCRYPTION_LEVELS);
+
+                        var currentEncryptionLevelString = helpers.ENCRYPTION_LEVELS[currentEncryptionLevel];
+
+                        if (currentEncryptionLevel >= desiredEncryptionLevel) {
+                            helpers.addResult(results, 0,
+                                `Connect instance is using ${currentEncryptionLevelString} for media streams encryption\
+                                which is greater than or equal to the desired encryption level ${config.desiredEncryptionLevelString}`,
+                                region, resource);
+                        } else {
+                            helpers.addResult(results, 2,
+                                `Connect instance is using ${currentEncryptionLevelString} for media streams encryption\
+                                which is less than the desired encryption level ${config.desiredEncryptionLevelString}`,
+                                region, resource);
+                        }
+                    } else {
+                        currentEncryptionLevel= 2; //awskms
```

Comment:
if we are setting the currentEncryptionLevel here this is not being checked after this. Is this okay?